lib/rust/ensogl/core/src/data/color/space/def.rs

```diff
@@ -389,6 +389,60 @@ impl Rgb {
         Self::new(r.into() / 255.0, g.into() / 255.0, b.into() / 255.0)
     }
 
+    /// Return a color if `css_hex` is a `#RGB` or `#RRGGBB` string, where `R`, `G`, `B` represent
```

Comment:
1. What is this syntax? ``` `css_hex` ```?
2. ``` if `css_hex` is a `#RGB` or `#RRGGBB` string ```, this sounds like the allowed strings are only "#RGB" and "#RRGGBB".
3. Sometimes you use backticks, sometimes not in this documentation - what is the reason for that?
4. "The format is adapted", (acording to dictionary, "adapted" = "to change behavior so that it is easier to function in a particular place or situation"). So how did you changed the format? It is not explained in this documentation.